src/Bicep.Core.UnitTests/TypeSystem/FunctionResolverTests.cs

```diff
@@ -122,7 +122,7 @@ object[] CreateRow(string returnedLiteral, string functionName, params string[]
 
             yield return CreateRow("IEZpenog", "base64", " Fizz ");
             yield return CreateRow(" Fizz ", "base64ToString", "IEZpenog");
-            yield return CreateRow("data:text/plain;charset=utf8;base64,IEZpenog", "dataUri", " Fizz ");
+            yield return CreateRow("data:text/plain;charset=utf-8;base64,IEZpenog", "dataUri", " Fizz ");
```

Comment:
Was this test failing after updating the expressions package? Both charset inputs should work, though `utf-8` is preferred.

Reply:
Ah in fact, I think it's just picking up your change in https://msazure.visualstudio.com/One/_git/AzureUX-Deployments/pullrequest/6041140